frontend/src/components/AddConfigModal/index.tsx

```
@@ -738,24 +751,67 @@ const AddConfigModal: React.FC<CustomConfigProps> = ({
+                        (newConfig.remote_port &&
+                          !portOptions.find(
+                            p => p.value === newConfig.remote_port,
+                          )) ? (
+                            <Input
+                              type='number'
+                              name='remote_port'
+                              value={newConfig.remote_port || ''}
+                              onChange={e => {
+                                const value = parseInt(e.target.value, 10)
+
+                                if (!isNaN(value)) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)